webapp/src/main/resources/public/js/components/workbench/SearchResults.jsx

```diff
@@ -656,6 +661,7 @@ let SearchResults = createReactClass({
                                 gap: "15px",
                             }}
                         >
+                            <div>Total text units: { filteredItemCount }</div>
```

Comment:
Could we conditionally render this in the rare event the `filteredItemCount` is null, to avoid rendering the div with no count ?

```suggestion
                            {filteredItemCount && <div>Total text units: { filteredItemCount }</div>}
```